plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/gettingstarted/GettingStartedAuthUtils.kt

```diff
@@ -229,13 +233,27 @@ fun reauthenticateWithQ(project: Project) {
 }
 
 fun emitUserState(project: Project) {
-    AuthTelemetry.userState(
-        project,
-        source = getStartupState().toString(),
-        authEnabledConnections = getEnabledConnections(project),
-        authStatus = getAuthStatus(project),
-        passive = true
-    )
+
+    val scopes = ToolkitAuthManager.getInstance().listConnections().flatMap { connection ->
+        when (connection) {
+            is ProfileSsoManagedBearerSsoConnection -> connection.scopes
+            is LegacyManagedBearerSsoConnection -> connection.scopes
+            else -> emptyList()
+        }
+    }.toSet()
```

Comment:
honestly it seems odd that we inspect every connection instead of what the extension is actively using, which seems more useful